app/controllers/permission_sets_controller.rb

```diff
@@ -68,6 +68,23 @@ def new_term
     authorize!(:update, @permission_set)
   end
 
+  def terms_api
+    # check for valid permission set
+    begin
+      permission_set = PermissionSet.find(params[:id])
+    rescue ActiveRecord::RecordNotFound
+      render(json: { "title": "Permission Set not found" }, status: 400) && (return false)
+    end
+    # check for terms on set
+    if permission_set.permission_set_terms.blank? || permission_set.permission_set_terms.last.inactivated_at.present?
```

Comment:
Is the order guaranteed for `last`?  Would it be better to check `permission_set.active_permission_set_terms`?

Reply:
Nvm, primary key is default.